R/bootstrap-layout.R

```diff
@@ -118,6 +120,7 @@ fluidRow <- function(...) {
 #' @param theme Alternative Bootstrap stylesheet (normally a css file within the
 #'   www directory). For example, to use the theme located at
 #'   `www/bootstrap.css` you would use `theme = "bootstrap.css"`.
+#' @param lang ISO 639-1 language code used within the shiny app. Default is set to empty value.
```

Comment:
It would help to explain with examples:

> ISO 639-1 language code for the HTML page, such as "en" or "ko". This will be used as the `lang` in the `<html>` tag, as in `<html lang="en">`. The default (NULL) results in an empty string.